doc/source/Tutorials/Scripting/QueryingAScene/index.md

```diff
@@ -1,296 +1,135 @@
-Querying A Scene
-================
+# Tutorial: Querying a Scene #
 
-Gaffer has a simple API for querying the scenes that are output from nodes. In this tutorial we'll see how to use this API to traverse a scene hierarchy and examine its state.
+Gaffer's scripting flexibility allows you to query the scene's parameters and objects as it flows through the node graph. In this tutorial, we will demonstrate how to use Gaffer's API to reference various scene parameters and traverse the scene hierarchy. We will cover the following:
```

Comment:
Let's not say "parameters" here - they're not a primary property of a scene. I really think we need to get this "Anatomy of a scene" concept nailed down as a guide before we go much further. I chose my words very carefully in the original text, but since my reasoning was opaque, it's going to represent a constant trap until we get it clarified.